src/main/java/by/mifort/automation/hr/dev/entity/Candidate.java

```diff
@@ -37,31 +36,31 @@
      *
      * @see CandidateUpdate
      */
-    @OneToMany(mappedBy = "candidate")
+    @OneToMany(mappedBy = "candidate", fetch = FetchType.LAZY)
     private List<CandidateUpdate> updates;
 
     /**
      * Keyboards that connection with candidate
      *
      * @see Keyword
      */
-    @OneToMany(mappedBy = "candidate")
+    @OneToMany(mappedBy = "candidate", fetch = FetchType.LAZY)
     private List<Keyword> keywords;
 
     /**
      * Communication history with candidate
      *
      * @see CommunicationHistory
      */
-    @OneToMany(mappedBy = "candidate")
+    @OneToMany(mappedBy = "candidate", fetch = FetchType.LAZY)
```

Comment:
look at source code. fetch is lazy by default

Reply:
fixed